Plugins/CGAL/vtkCGAL/vtkCGALEfficientRANSAC.cxx

```diff
@@ -0,0 +1,277 @@
+/**
+* \class vtkCGALEfficientRANSAC
+* 
+* \brief todo
+* 
+* 
+* 
+* Inputs: todo
+* Output: todo
+* 
+*/
+
+#include <vtkCGALEfficientRANSAC.h>
+
+// -- VTK
+#include <vtkCellData.h>
+#include <vtkCGALUtilities.h>
+#include <vtkFieldData.h>
+#include <vtkFloatArray.h>
+#include <vtkInformation.h>
+#include <vtkInformationVector.h>
+#include <vtkIntArray.h>
+#include <vtkPointData.h>
+#include <vtkSmartPointer.h>
+#include <vtkTimerLog.h>
+
+// -- CGAL
+#include <CGAL/property_map.h>
+#include <CGAL/Point_with_normal_3.h>
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Shape_detection/Efficient_RANSAC.h>
+
+// Type declarations.
+typedef CGAL::Exact_predicates_inexact_constructions_kernel                     Kernel;
+typedef Kernel::FT                                                              FT;
+typedef CGAL::First_of_pair_property_map<vtkCGALUtilities::Point_with_normal0>  Point_map;
+typedef CGAL::Second_of_pair_property_map<vtkCGALUtilities::Point_with_normal0> Normal_map;
+
+typedef CGAL::Shape_detection::Efficient_RANSAC_traits
+<Kernel, vtkCGALUtilities::Pwn_vector0, Point_map, Normal_map>  Traits;
+typedef CGAL::Shape_detection::Efficient_RANSAC<Traits>         Efficient_ransac;
+typedef CGAL::Shape_detection::Plane<Traits>                    Plane;
+
+// -----------------------------------------------------------------------------
+// Declare the plugin
+vtkStandardNewMacro(vtkCGALEfficientRANSAC);
+
+// -----------------------------------------------------------------------------
+// Constructor
+// Todo.
+vtkCGALEfficientRANSAC::vtkCGALEfficientRANSAC()
+{
+  this->NumberOfIterations = 1;
+  this->UseParameters = 0;
+  this->MinPoints = 200;
+
+  this->Probability = 0.05;
+  this->Epsilon = 0.002;
+  this->ClusterEpsilon = 0.01;
+  this->MaxNormalDeviation = 0.9;
+}
+
+// -----------------------------------------------------------------------------
+// Todo.
```

Comment:
What;'s TODO?

Reply:
Description of the method, a.k.a. what is done in the RequestData of this filter